ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/StringTemplateRule.kt

```diff
@@ -41,13 +46,15 @@ class StringTemplateRule : Rule("string-template") {
                 ) {
                     emit(dot.startOffset, "Redundant \"toString()\" call in string template", true)
                     if (autoCorrect) {
+                        entryExpression = (entryExpression as? KtDotQualifiedExpression)?.receiverExpression
```

Comment:
What is this line for?

Reply:
If this line is not exists, `testFormat` will fail because `entryExpression` is KtDotQualifiedExpression.

`callExpression` of `dotQualifiedExpression` will be removed in the next line, so we save `receiverExpression` in `entryExpression`.